draft-ietf-spice-sd-cwt.md

```diff
@@ -751,51 +709,53 @@ Reference: RFC XXXX
 
 IANA is requested to add the following entries to the CWT claims registry (https://www.iana.org/assignments/cwt/cwt.xhtml).
 
-### sd_alg
+<!-- https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt-12#name-json-web-token-claims-regis -->
```

Comment:
??

Reply:
I think my intention was to relate the registration requests... This should probably just be removed.